test/unit/lodash.spec.js

```diff
@@ -0,0 +1,475 @@
+            expect(params).to.have.length(1);
+        });
+        it('should not contain proper11111111111111111111111', function() {
+            var obj = params[0];
+            for (var key in obj) {
+                if (obj.hasOwnProperty(key)) {
+                    // co tu wpisac ...
+                }
+            }
+        });
+        it('should return array property names of functions', function() {
+            expect(_.functionsIn.apply(_, params)).to.eql(['x', 'z']);
+        });
+    });
+
+    describe('pick', function() {
+        beforeEach(function() {
+            params = datasets.pick();
+        });
+        it('should match types of passing elements', function() {
+            var element = params[0] instanceof Object;
+            var element1 = params[1] instanceof Object;
+            expect(element).to.eql(true);
+            expect(element1).to.eql(true);
+            expect(params.length).to.eql(2);
+        });
+        it('should pick red and yellow apples', function() {
+            expect(_.pick.apply(_, params)).to.eql({
+                'red': 6,
+                'yellow': 2
+            });
+        });
+    });
+
+    describe('pick1', function() {
+        beforeEach(function() {
+            params = datasets.pick1();
+        });
+        it('should match types of passing elements', function() {
+            var element = params[0] instanceof Object;
+            var element1 = typeof params[1];
+            expect(element).to.eql(true);
+            expect(element1).to.eql('string');
+            expect(params.length).to.eql(2);
+        });
+        it('should pick green apples', function() {
+            expect(_.pick.apply(_, params)).to.eql({
+                'green': 4,
+            });
+        });
+    });
+
+    describe('pickBy', function() {
+        beforeEach(function() {
+            params = datasets.pickBy();
+        });
+        it('should match types of passing elements', function() {
+            var element = params[0] instanceof Object;
+            var element1 = params[1] instanceof Function;
+            expect(element).to.eql(true);
+            expect(element1).to.eql(true);
+            expect(params.length).to.eql(2);
+        });
+        it('should pick ', function() {
+            expect(_.pickBy.apply(_, params)).to.eql({
+                'x': 16,
+            });
+        });
+    });
+
+    describe('set', function() {
+        beforeEach(function() {
+            params = datasets.set();
+        });
+        it('should match types of passing elements', function() {
+            var element1 = params[0] instanceof Object;
+            var element2 = typeof params[1];
+            expect(element1).to.eql(true);
+            expect(element2).to.eql('string');
+            expect(params.length).to.eql(3);
+        });
+        it('should set height value of rect1 to 8', function() {
+            expect(_.set.apply(_, params)).to.eql({
```

Comment:
Again, this is quite an open test case and you may make it more specified I think. Right know if I am right I would be able to bypass it with some values.